idaes/ui/flowsheet.py

```diff
@@ -380,6 +381,24 @@ def _construct_output_json(self):
         self._construct_jointjs_json()
 
     def _construct_model_json(self):
+        from idaes.core.util.tables import create_stream_table_dataframe # deferred to avoid circular import
+
+        # Get the stream table and add it to the model json
+        self._stream_table_df = create_stream_table_dataframe(self.arcs)
+        
+        # Change the index of the pandas dataframe to not be the variables
+        self._stream_table_df = self._stream_table_df.reset_index().rename(columns={"index": "Variable"})
+        self._stream_table_df = self._stream_table_df.reset_index().rename(columns={"index": ""})
```

Comment:
In pandas this type of operations is often written in a "fluent" style, i.e.:

```python        
	self._stream_table_df = (
		create_stream_table_dataframe(self.arcs)
        	# Change the index of the pandas dataframe to not be the variables
		.reset_index().rename(columns={"index": "Variable"})
		.reset_index().rename(columns={"index": ""})
	)
```

Personally I find this more readable, but as they say, readability is in the eye of the beholder :)

Reply:
It does look better